test.js

```diff
@@ -3,7 +3,23 @@ const rewire = require('rewire');
 
 const bot = rewire('./test_compiled_bot.js');
 let utilities = bot.__get__("utilities");
-let SPECS = bot.__get__("SPECS")
+let pilgrim = bot.__get__("pilgrim");
+let SPECS = bot.__get__("SPECS");
+
+generate_grid = (x, y, val = false) => {

```

Comment:
Is this useful for generating a map for testing?

Reply:
That's the idea. It was getting pretty messy hardcoding all those 2d list of lists in each and every test.

Reply:
this is good .. helps to write other tests also.Thanks